Makefile

```
@@ -169,15 +210,19 @@ down: ## Stop the docker hub
+	@echo "Connecting to core-service PHP container..."
 	@$(EXEC_PHP) sh
 
 logs: ## Show all logs
-	@$(DOCKER_COMPOSE) logs --follow
+	@$(DOCKER_COMPOSE) logs
 
 new-logs: ## Show live logs
 	@$(DOCKER_COMPOSE) logs --tail=0 --follow
 
-start: up ## Start docker
+start: up build-k6-docker ## Start docker with k6
```

checkmake (0.2.2)

[warning] Target "start" should be declared PHONY.

(phonydeclared)